srcCpp/Infrastructure_micro.h

```diff
@@ -54,6 +60,10 @@ class PerftestClock {
     RTI_INT32 clockSec;
     RTI_UINT32 clockUsec;
 
+#if defined(RTI_WIN32)
+    double PCFreq = 0.0;
```

Comment:
Could you add a more descriptive name? Also, add some documentation here and in the .cxx explaining that for Micro on Windows the resolution of the clock is not good enough and we will just use the native api.

Reply:
Done